watchdog/e2e-tests/utils.sh

```diff
@@ -2,24 +2,7 @@
 
 # Function to deploy the watchdog canister for mainnet bitcoin_canister.
 deploy_watchdog_canister_mainnet() {
-  BITCOIN_NETWORK=mainnet
-  BITCOIN_CANISTER_ID=ghsi2-tqaaa-aaaan-aaaca-cai
-  dfx deploy --no-wallet watchdog --argument "(opt record {
-    bitcoin_network = variant { ${BITCOIN_NETWORK} };
-    blocks_behind_threshold = 2;
-    blocks_ahead_threshold = 2;
-    min_explorers = 2;
```

Comment:
This can still be a problem for e2e-tests, since dfx of newer versions do not support IPv4.
See context in https://github.com/dfinity/bitcoin-canister/pull/270

Reply:
I addressed with rust features, so that we don't reduce it in production as well.

Reply:
nice! thank you.